django/thunderstore/core/settings.py

```diff
@@ -83,6 +84,10 @@
 SECRET_KEY = env.str("SECRET_KEY")
 
 ALLOWED_HOSTS = env.list("ALLOWED_HOSTS")
+CORS_ALLOWED_ORIGINS = env.list("CORS_ALLOWED_ORIGINS")
```

Comment:
What happens if this is not defined to anything? AKA will the production API break with this change if the variable is undefined?

Reply:
It will default to `[]` resulting in no allowed origins (other than the target's origin). Testing in a staging environment might help check, but it should work as expected. We don't currently have a CORS header so there won't be a breaking change.